packages/react/src/primitives/Rating/RatingIcon.tsx

```diff
@@ -15,11 +15,9 @@ export const RatingIcon: React.FC<RatingIconProps> = ({
   className,
 }) => {
   return (
-    <View as="span" className={classNames(`amplify-rating-icon-container`)}>
-      <View as="label" className={classNames(`amplify-rating-label`)}>
```

Comment:
not sure what's the `label` for, so removed it

Reply:
I believe the label tag was a remnant of earlier designs when this component was not read only.  Looking at some other rating components out there they use a label and hidden checkbox to mark user selections.  @reesscot brought up a good point that we could have customers who are targeting this label for styling and so if we don't need to remove it, then it would be best to leave it in.  @0618 other than the extra clutter, was the `label` causing any problems? If not then I agree that we should leave it in as to not create a potentially breaking change for customers.

Reply:
@jacoblogan yes, "empty label" is a type of a11y error, so I removed it but kept the class selector in this PR. Is that ok?

Reply:
Makes sense. If it's causing a11y error let's remove it.